File: 20141214-todoapp-javascript/routes/todoRepository.js

var datas = [ {
	title : "task 1 from backend",
	completed : false
}, {
	title : "task 2 from backend",
	completed : false
} ];

exports.findAll = function() {
	return datas;
};
exports.updateAll = function(newDatas) {
	datas = newDatas;
};

File: 20141215-todoapp-java/src/main/java/com/dfremont/blog/TodoRepository.java
package com.dfremont.blog;

public class TodoRepository {

	// datas
	private static TodoModel[] todos = new TodoModel[] { //
	//
			new TodoModel("task 1 from backend", false), //
			new TodoModel("task 2 from backend", false) };

	// methods

	public TodoModel[] findAll() {
		return todos;
	}

	public void updateAll(TodoModel[] updatedTodos) {
		todos = updatedTodos;
	}

}
